test/fgpu/test_engine.py

```diff
@@ -933,6 +939,61 @@ async def test_missing_heaps(
             == np.sum(~dst_present) * n_substreams
         )
 
+        # Sensor is not present in the narrowband mode.
+        if output.decimation == 1:
+            # Information needed to check that the dig-rms-dbfs sensor is correctly
+            # going into FAILURE when there are missing samples.
+            class Update(Enum):
+                NORMAL = 1
+                FAILURE = 2
+                OPTIONAL_FAILURE = 3
+
+            expected_updates = []
+            # The sensor is updated once per out_item.
+            spectra_per_output_chunk = engine_server.chunk_jones // output.channels
+            heaps_per_output_chunk = spectra_per_output_chunk // spectra_per_heap
```

Comment:
Should be batches_per_output_chunk.

Reply:
We did discuss this and I started making the change, but I wondered about the consistency.

There are other local variables used in the function such as `total_heaps` and `spectra_per_heap` which should probably also be referred to as batches?